README.md

```diff
@@ -8,7 +8,7 @@
 
 * 💬 **Chat** Nextcloud Talk comes with a simple text chat, allowing you to share or upload files from your Nextcloud Files app or local device and mentioning other participants.
 * 👥 **Private, group, public and password protected calls!** Invite someone, a whole group or send a public link to invite to a call.
-* 💻 **Screen sharing!** Share your screen with participants of your call. You just need to use Firefox version 52 (or newer), latest Edge or Chrome 49 (or newer) with this [Chrome extension](https://chrome.google.com/webstore/detail/screensharing-for-nextclo/kepnpjhambipllfmgmbapncekcmabkol).
+* 💻 **Screen sharing!** Share your screen with participants of your call. You just need to use Firefox version 66 (or newer), latest Edge or Chrome 49 (or newer) with this [Chrome extension](https://chrome.google.com/webstore/detail/screensharing-for-nextclo/kepnpjhambipllfmgmbapncekcmabkol).
```

Comment:
is the chrome extension even needed any more ?

Reply:
On old versions yes, but I don't know with which version it started working, maybe @danxuliu knows?

Reply:
The extension is no longer needed since Chromium 72 (when [support for `getDisplayMedia` was introduced](https://www.caniuse.com/?search=getdisplaymedia)).

Reply:
Thanks, updated readme and info.xml